File: applications/indigo/src/search/documents/characters.go
package documents

type Media struct {
	Id        string `json:"id"`
	Thumbnail string `json:"thumbnail"`
	Title     string `json:"title"`
}

type Character struct {
	Id        string `json:"id"`
	Thumbnail string `json:"thumbnail"`
	Name      string `json:"name"`
	Media     Media  `json:"media"`
}

File: applications/medusa/src/client/components/FullPost/components/info/sections/characters/Characters.js
/**
 * @flow
 */
import type { Node } from 'react'
import Element
  from '../../../../../Element/Element'
import { Wrap } from '@chakra-ui/react'

type Props = {
  characters: {
    key: {
      id: string,
      name: string,
      media: {
        id: string,
        title: string
      }
    }
  },
};

export default function Characters ({ characters }: Props): Node {
  return (
    <Wrap justify='center'>
      {Object.keys(characters).map(item => (
        <Element
          key={characters[item].id}
          selected={false}
          title={characters[item].name}
          subheader={characters[item].media.title}
          thumbnail={characters[item].thumbnail}
        />
      ))}
    </Wrap>
  )
}
